csrc/fused_moe/megamoe/tests/README.md

```
@@ -0,0 +1,45 @@
+
+```bash
+nvcc -std=c++17 -arch=sm_90a -O2 \
+     -I vllm/csrc/moe/moe_monokernel/src \
+     vllm/csrc/moe/moe_monokernel/tests/mbarrier_sanity_test.cu \
+     -o /tmp/mbarrier_sanity_test
+/tmp/mbarrier_sanity_test
+```
+
+Expected output on an H100 / H200:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)